### PR TITLE
crop

### PR DESCRIPTION
**Pull request by kashike**

### CODE REVIEW

**Comment by kashike** [comment]
crop

File: docs/block/action/fertilize.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "with": [
+        {
+            "item": "minecraft:dye",
```
Comment (Zidane) [inline docs/block/action/fertilize.json]: We should include a note that 1.13 will break this and data will no longer be available so consider this already deprecated.

File: src/main/java/com/almuradev/almura/content/type/block/BlockType.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import com.almuradev.almura.content.type.block.type.normal.NormalBlock;
+import com.almuradev.almura.content.type.block.type.normal.state.NormalBlockStateDefinitionBuilder;
+
+import java.util.Locale;
+import java.util.function.Supplier;
+
+import javax.annotation.Nullable;
+
+/**
+ * An enumeration of supported block types.
```
Comment (Zidane) [inline src/main/java/com/almuradev/almura/content/type/block/BlockType.java]: We have Asset.Type, why is this needed?

File: src/main/java/com/almuradev/almura/content/type/block/component/action/fertilize/FertilizeAction.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+
+import java.util.Random;
+
+public class FertilizeAction implements Action {
+
+    @Override
+    public void apply(final EntityPlayer player, final Block block, final BlockPos pos, final Random random, final ItemStack stack) {
```
Comment (Zidane) [inline src/main/java/com/almuradev/almura/content/type/block/component/action/fertilize/FertilizeAction.java]: Not sure these action classes will work, so long as we understand how fertilize works.

Basically there is a variable amount of fertilizer needed to advance the age of the crop in the old legacy files (meaning some crops may need 1 bonemeal or 3 to advance, 1-3 chance). Would the action track this or the block listener?

**Comment by kashike** [comment]
#372
